scripts/ci-helper.sh

```diff
@@ -134,6 +134,8 @@ function prepare_legal() {
   mkdir -p ".release/linux/package/usr/share/doc/$PKG_NAME"
   cp dist/EULA.txt ".release/linux/package/usr/share/doc/$PKG_NAME/EULA.txt"
   cp dist/TermsOfEvaluation.txt ".release/linux/package/usr/share/doc/$PKG_NAME/TermsOfEvaluation.txt"
+  mkdir -p ".release/linux/package/usr/share/doc/$PKG_NAME"
```

Comment:
@alvin-huang, I used the package name here instead of the binary name you suggested in the previous comment because of how the EULA and Terms of Evaluation also use the package name. Since I'm not 100% familiarized with the packaging system of Vault, I'm looking forward to hearing your opinion on whether or not this looks right.

Reply:
I'm also a bit confused from where exactly the License will come from -- is this going to be an env variable set somewhere?

Reply:
FYI @ryancragun as well

Reply:
This line is unnecessary because we've already created the same directory on line 134. We also haven't defined a `$LICENSE_DIRECTORY` variable on the subsequent line, so that will almost certainly do nothing for us.